py/private/py_pex_binary.bzl

```diff
@@ -0,0 +1,158 @@
+"Create python zip file https://peps.python.org/pep-0441/ (PEX)"
+
+load("@rules_python//python:defs.bzl", "PyInfo")
+load("//py/private:py_semantics.bzl", _py_semantics = "semantics")
+load("//py/private/toolchain:types.bzl", "PY_TOOLCHAIN")
+
+def _runfiles_path(file, workspace):
+    if file.short_path.startswith("../"):
+        return file.short_path[3:]
+    else:
+        return workspace + "/" + file.short_path
+
+exclude_paths = [
+    "toolchain",
+    "aspect_rules_py/py/tools/",
+    "rules_python~~python~",
```

Comment:
> `rules_python~~python~`

Will this path change if we set `--incompatible_use_plus_in_repo_names`? It's going to be set to true by default in bazel 8.

context: https://github.com/bazelbuild/bazel/issues/23127

Reply:
Yes, this will stop working.

Reply:
Sent a filter for plus version as well. LMK if that does it.